reactDashboard/src/components/DataCharts/DateRangeSlider.jsx

```diff
@@ -0,0 +1,147 @@
+import React, { useState, useEffect } from "react";
+import { makeStyles, withStyles } from "@material-ui/core/styles";
+import Slider from "@material-ui/core/Slider";
+import moment from "moment";
+import { getNhsCsvDataDateRange } from "../Utils/CsvUtils";
+import "./DataCharts.css";
+
+const useStyles = makeStyles({
```

Comment:
There seems to be two layers of styling here - can they be combined into one?

Reply:
moved to another ticket